numba/core/pythonapi.py

```diff
@@ -944,13 +944,17 @@ def call_method(self, callee, method, objargs=()):
         return self.builder.call(fn, args)
 
     def call(self, callee, args=None, kws=None):
+        args_was_none = args is None
         if args is None:
```

Comment:
Since we only support Python 3.8+, I think this might be an appropriate place for a walrus to avoid repetition of the expression:

```suggestion
        if args_was_none := args is None:
```